randobot/zsr.py

```diff
@@ -129,6 +138,8 @@ def roll_seed(self, preset, encrypt, dev, settings, race_type):
             params['encrypt'] = 'true'
         if encrypt and dev:
             params['locked'] = 'true'
+        if password and not dev:
```

Comment:
Why is the password lock feature limited to seeds rolled on release?

Reply:
The API allows this, but i am very torn. It seems like a perfect way to give people a false sense of security when it is somewhat easy to circumvent.

Reply:
I understand the concern, but imo having a password that's somewhat easy to circumvent is still better than no password at all.

Reply:
Fair enough, lets add it.